scripts/forseti-client/forseti_client_startup_script.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+exec > /tmp/deployment.log
+exec 2>&1
+
+# Ubuntu update.
+sudo apt-get update -y
+sudo apt-get upgrade -y
+
+# Forseti setup.
+sudo apt-get install -y git unzip
+
+# Forseti dependencies
+sudo apt-get install -y libffi-dev libssl-dev libmysqlclient-dev python-pip python-dev build-essential
+
+USER=ubuntu
+USER_HOME=/home/ubuntu
+
+# Install fluentd if necessary.
+FLUENTD=$(ls /usr/sbin/google-fluentd)
+if [ -z "$FLUENTD" ]; then
+    cd $USER_HOME
+    curl -sSO https://dl.google.com/cloudagents/install-logging-agent.sh
+    bash install-logging-agent.sh
+fi
+
+# Install Forseti Security.
+cd $USER_HOME
+rm -rf *forseti*
+
+# Download Forseti source code
+git clone ${forseti_repo_url}
```

Comment:
Instead of templating out scripts, can we provide a config file that gets sourced or provide CLI arguments? On my first pass of this review I thought this was using bash interpolation syntax, and others could be surprised by this.